<launch>
    <!-- args --> 
    <!-- Behzad's workstation ip --> 
    <!-- arg name="host_ip" value="10.157.34.101" /--> 
    <arg name="host_ip" value="10.157.90.51" /> 
    <!-- Hasans's workstation ip --> 
    <!-- arg name="host_ip" value="10.157.34.208" /--> 

    <!-- setting DEBUG to false prevents publishing and subscribing to rviz /--> 
    <!-- also planning to prevent stdio stuff(if they really matter)--> 
    <arg name="DEBUG" value="true" /> 
    <param name="ros_DEBUG" type="bool" value="$(arg DEBUG)" />


    <!-- obj trackingnode --> 
    <param name="ip_addr" type="string" value="$(arg host_ip)" />
     
    <node pkg="follow_the_leader" type="tracking_continuous" name="tracking_continuous" output="screen">
    <param name="max_n_track_before_det_count" type="int" value="30" />
    <param name="tracking_threshold" type="double" value=".8" />
    <param name="min_allowed_tracking_treshold" type="double" value=".6" />
    <param name="max_strike" type="int" value="10" />
    
    <!-- param name="mav_name" type="string" value="$(arg mav_name)" / -->
    </node>
    <comment launch-prefix="gdb -ex run --args"/>



    <node pkg="follow_the_leader" type="buffer_track_node" name="buffer_track_node" output="screen">
    <param name="max_n_track_before_det_count" type="int" value="30" />
    <param name="tracking_threshold" type="double" value=".8" />
    <param name="min_allowed_tracking_treshold" type="double" value=".6" />
    <param name="max_strike" type="int" value="10" />
    
    <!-- param name="mav_name" type="string" value="$(arg mav_name)" / -->
    </node>
    <comment launch-prefix="gdb -ex run --args"/>




</launch>
